chemicaltoolbox/rdkit/sucos.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
```

Comment:
should the sucos tool be in its own subfolder?

Reply:
moved to sucos dir.

Reply:
I think it should be one level up rather than in the rdkit folder though

Reply:
It's based heavily on RDKit.